packages/wonder-blocks-modal/components/modal-context.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import * as React from "react";
+
+// babel-plugin-strip-flow-types doesn't know how to deal with
+// React.createContext<Context>(); where Context is a flow type.
```

Comment:
I think maybe we can solve this with other idioms to help Flow understand the type? In Test Prep's `global-exam-data-context.js`, we make a type assertion about the value that we pass to `React.createContext`:

```
const GlobalExamDataContext = React.createContext((null: ?Exam));
```

I think this gets Flow to realize that it's a `React.Context<?Exam>`.

Reply:
Cool.  I'll give this a try.  I hope someone updates the `strip-flow-types` plugin eventually though.

Reply:
Oh yeah, this definitely works - I had it work in my `<Layout>` PR. I had luck in doing:

```
type ContextType = {closeModal: ....type....};

export default React.createContext({closeModal: undefined}: ContextType);

```

You can see an example of how I did it here:
https://github.com/Khan/wonder-blocks/pull/311/files#diff-545f77fd7a4db77f9e1d373132cca3f9